azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/executingflowtriggerspage.vm

```diff
@@ -175,9 +175,9 @@
               #end
 
               <td>
-                <button type="button" id="showTriggerProperties" class="btn btn-danger btn-sm"
+                <button type="button" id="showTriggerProperties" class="btn btn-sm btn-info"
```

Comment:
Why do you want to change the button type from info to danger? 
`.btn-danger | Indicates a dangerous or potentially negative action`
It seems to me triggerProperties are just info not some dangerous action.